generate_so.py

```diff
@@ -514,4 +515,133 @@ def _make(dirpath: str) -> None:
         version='0.1',
         create_date='2019-07-21T12:15:54Z',
     )
+    _make('out/ssop')
+    _make('out/ssop/pkg')
```

Comment:
Shouldn't `_make` create directories recursively?

Reply:
True, since it uses `os.makedirs` under the hood.